src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -316,6 +316,11 @@ private static bool TryGetGrpcMethod(Activity activity, out string grpcMethod)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static void AddGrpcAttributes(Activity activity, string grpcMethod, HttpContext context)
         {
+            // The RPC semantic conventions indicate the span name
+            // should not have a leading forward slash.
+            // https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#span-name
+            activity.DisplayName = grpcMethod.Trim('/');
```

Comment:
`TrimStart` might be slightly faster (if we're only concerned with the leading character).

Reply:
Good idea. Trimming only leading character should be ok in this context since `grpcMethod` is really just derived from the `HttpContext.Request.Path`.